sqlstore/src/main/resources/db/migration/V8900__hibernatize_everything.sql

```diff
@@ -1,3 +1,13 @@
 UPDATE KitDescriptor SET kitType = UPPER(kitType), platformType = UPPER(platformType);
 
-UPDATE Platform SET name = UPPER(name);
\ No newline at end of file
+UPDATE Platform SET name = UPPER(name);
+
+ALTER TABLE SequencerReference ADD COLUMN ip VARCHAR(50) NOT NULL DEFAULT 'localhost';
+-- H2 doesn't have INET_NTOA function
+-- StartNoTest
+UPDATE SequencerReference tabl SET ip = (
```

Comment:
You should be able to do just `UPDATE SequencerReference SET ip = INET_NTOA(ipAddress);`